packaging/docker/build.sh

```diff
@@ -60,7 +60,7 @@ docker run --rm --privileged multiarch/qemu-user-static:register --reset
 
 # Build images using multi-arch Dockerfile.
 for ARCH in ${ARCHS[@]}; do
-     TAG="${REPOSITORY}:${VERSION}-${ARCH}"
+     TAG="${REPOSITORY,,}:${VERSION}-${ARCH}"
```

Comment:
:rofl:  gotta love bash — how's one meant to search for what `,,` means.

Reply:
Not hard in the manpages, but yeah, it's unnecessarily terse. The funny thing is that the equivalent for uppercase is `${foo^^}`. I guess the commas are supposed to look like downwards pointing arrows?